app/android/app/build.gradle

```diff
@@ -36,7 +36,23 @@ if (flutterVersionName == null) {
 android {
     compileSdkVersion 31
 
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+
+    sourceSets {
+        main.java.srcDirs += 'src/main/kotlin'
+    }
```

Comment:
If you were wondering where I get this from: I copied this from the Flutter counter app when you create a new Flutter app with `flutter create app` (I used Flutter 2.10.5).